appveyor.yml

```diff
@@ -21,14 +37,16 @@ for:
       ./ninja all\n
       ./ninja_test 2>&1\n
       ./misc/ninja_syntax_test.py 2>&1\n\"@"
-  -
-    matrix:
+
+  - matrix:
```

Comment:
Is this necessary?

Reply:
As in, will it break something if the change isn't made? No, it won't break anything if that change is not made.

However, it is necessary for my own sanity.

You can edit it back out if you like after the PR is merged.